code/modules/reagents/chems/chems_medicines.dm

```diff
@@ -104,7 +104,7 @@
 	if(remove_generic)
 		ADJ_STATUS(M, STAT_DROWSY, -6 * removed)
 		M.adjust_hallucination(-9 * removed)
-		M.add_up_to_chemical_effect(CE_ANTITOX, 1)
```

Comment:
why this?

Reply:
I removed the proc. It wasn't working properly anyway.